composer.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "sensiolabs/typescript-bundle",
+    "description": "Typescript support for Symfony + AssetMapper",
```

```suggestion
    "description": "TypeScript support for Symfony + AssetMapper",
```

And this needs to be changed in a few places (it's correct already in most places). Once upon a time, Fabien corrected me on Javascript -> JavaScript 100 times, so now it's burned into my eyes :)